Python/pywarpx/picmi.py

```diff
@@ -598,6 +598,9 @@ def init(self, kw):
         self.potential_zmax = kw.pop('warpx_potential_hi_z', None)
         self.reflect_all_velocities = kw.pop('warpx_reflect_all_velocities', None)
 
+        self.warpx_start_moving_window_step = kw.pop('warpx_start_moving_window_step', 0)
```

Comment:
The default values here should be `None`, as is done for the other parameters. The default values of these parameters (i.e. 0 and -1) are already set in the c++ and shouldn't be duplicated here.

Reply:
Thanks for your comment, and i had change them to be None. Also, i add the description of two new parameters to the triple quoted doc strings.